projects/plugins/wpcomsh/tests/imports/Playground_Clean_Up_Test.php

```diff
@@ -17,10 +17,12 @@ class Playground_Clean_Up_Test extends WP_UnitTestCase {
 	 */
 	public function test_error_open_an_not_existing_file() {
 		$tmp_folder = tempnam( sys_get_temp_dir(), uniqid() );
-
-		Playground_Clean_Up::remove_tmp_files( uniqid(), $tmp_folder );
-
-		$this->assertFalse( is_dir( $tmp_folder ) );
+		try {
+			Playground_Clean_Up::remove_tmp_files( uniqid(), $tmp_folder );
```

Comment:
But what if `uniqid()` turns up a real file? 😱 😄

On a more serious note both for this call and the one a few dozen lines down:
* What makes us need the try/catch?
* Why don't we move the handling into `Playground_Clean_Up::remove_tmp_files`?
* Should the assertions be outside the try/catch?

Reply:
> But what if `uniqid()` turns up a real file? 😱 😄

Preexisting issue. 😀 I didn't dig into this to see what it's actually testing, I just added some cleanup code for the file it does create.

> What makes us need the try/catch?

It's a try/finally. The finally block makes sure the unlink happens even if the call throws or the assertion fails.

> Why don't we move the handling into `Playground_Clean_Up::remove_tmp_files`?

🤷 Apparently that's not supposed to delete the second arg if it's not a directory? I haven't looked at it.

> Should the assertions be outside the try/catch?

Probably doesn't matter.